mathics/doc/latex_doc.py

```diff
@@ -899,15 +907,23 @@ def latex(self, doc_data: dict, quiet=False, chapters=None) -> str:
             print(f"    Formatting Subsection Section {self.title}")
 
         title = escape_latex(self.title)
+
         if self.operator:
-            code_str = get_latex_operator(self.operator)
+            code_str = get_latex_operator_enclosed(self.operator)
             title += f" ({code_str})"
         index = (
             r"\index{%s}" % escape_latex(self.title)
             if self.chapter.part.is_reference
             else ""
         )
         content = self.doc.latex(doc_data)
+        # This just replaces the occurrences of the unicode
+        # character associated to the operator.
+        # At some point we would like to scan all the unicode characters
+        # and replace them according to the context.
+        if self.operator and not self.operator.isascii():
```

Comment:
This replaces all the ocurrences of the operator in its unicode form, by its LaTeX form. Eventually we would like to translate any unicode character appearing in the docstring. Notice also that at this point, the documentation contains both the test cases and their results.

Reply:
Eventually, we'd like to ditch all of this.

Reply:
> Eventually, we'd like to ditch all of this.

Even better!